src/main/java/aws/retrospective/controller/CommentController.java

```diff
@@ -0,0 +1,55 @@
+package aws.retrospective.controller;
+
+import aws.retrospective.entity.Comment;
+import aws.retrospective.service.CommentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api/comments")
+public class CommentController {
+
+    private final CommentService commentService;
+
+    @Autowired
+    public CommentController(CommentService commentService) {
+        this.commentService = commentService;
+    }
```

Comment:
```suggestion
@RequiredArgsConstructor
public class CommentController {

    private final CommentService commentService;
```

다른 클래스 보니 생성자는 롬복 어노테이션으로 대체하고 있네요. 애매할 땐 미리 만들어진 코드를 보고 규칙을 유추하실 수 있습니다. 규칙을 지키면 뒤에 작업하시는 분들에게도 혼란을 줄일 수 있어요~!